spec/cellular/backends/cool_sms_spec.rb

```diff
@@ -14,6 +14,19 @@
     }
   }
 
+  let(:payload){
```

Comment:
Ah, my bad, was supposed to say `around { and before }`:

```
list.each { |item|
  # ..
}
```

No trailing spaces :)